inc/agentmodule.class.php

```diff
@@ -449,7 +449,8 @@ static function getUrlForModule($modulename, $entities_id = -1) {
     */
    static function getModules() {
       $a_modules = [];
-      $a_data = getAllDatasFromTable(PluginFusioninventoryAgentmodule::getTable());
+      $dbUtils = new DbUtils();
```

Comment:
This change is not mandatory. In facts, It should stay as it for now; until core provides some dependency injection.